plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/catalog/AbstractTrinoCatalog.java

```diff
@@ -85,19 +91,22 @@
     private final CatalogName catalogName;
     private final TypeManager typeManager;
     protected final IcebergTableOperationsProvider tableOperationsProvider;
+    protected final TrinoFileSystemFactory fileSystemFactory;
```

Comment:
`private`?

Reply:
`fileSystemFactory` is being used in `TrinoHiveCatalog` and `TrinoGlueCatalog`